recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
 package:
   name: conda-forge-build-setup
-  version: 1.0.1
+  version: 1.0.2
```

Comment:
Could we actually do 1.1.0? Was thinking the last number would represent pinning changes. The middle number represents package changes (bug fixes). The first number would represent breaking package changes.

Though we haven't actually agreed on these per se. It seems helpful to understand what has changed with this package if something goes wrong. This is one way we might do this, but would be open to other suggestions.